app/design/frontend/Jungle-luma/jungle_theme/composer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Jungle-luma/jungle_theme",
+    "description": "N/A",
+    "type": "magento2-theme",
+    "version": "1.0.0",
+    "license": [
+        "OSL-3.0",
+        "AFL-3.0"
+    ],
+    "config": {
+        "sort-packages": true
+    },
+    "version": "100.4.5",
+    "require": {
+        "php": "~7.4.0||~8.1.0",
+        "magento/framework": "103.0.*",
+        "magento/theme-frontend-blank": "100.4.*"
```

Comment:
Your theme is inherited from luma, why did you add dependency to the blank, change it

Reply:
17. made changes to "magento/theme-frontend-luma".